docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md

```diff
@@ -312,7 +314,20 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
 
         **Tip:** We recommend that you provide a specific default value of the `title`.
         Use a title that describes the service instance so that the end user knows the specific purpose of the service instance.
+        
```

Comment:
Remove this part. I don't know if it is needed or not.

Reply:
I am not sure which part you refer to. Title, Description, etc. are different than apiInfo.title, apiInfo.description that were replaced.